praw/__init__.py

```diff
@@ -1610,9 +1610,8 @@ def set_settings(self, subreddit, title, public_description='',
                      submit_link_label='', submit_text_label='',
                      exclude_banned_modqueue=False, comment_score_hide_mins=0,
                      public_traffic=False, collapse_deleted_comments=False,
-                     spam_comments='low', spam_links='high',
-                     spam_selfposts='high', submit_text='',
-                     hide_ads=False, **kwargs):
+                     spam_comments='low', spam_links='high', spam_selfposts='high',
```

Comment:
These lines are too long. They need to be changed back to their original length in order for travis to pass (or shorter, make new lines if necessary.)